src/profile/helpers/formik.ts

```diff
@@ -51,3 +51,24 @@ export function getFieldError<FormValues>(
     t(message, options);
   return getError<FormValues>(formikProps, fieldName, renderError, isOldData);
 }
+
+export function createFormFieldHelpers<FormikValues>(
+  t: TFunction,
```

Comment:
In general I think variables with only 1 letter should be avoided, since they are not very descriptive and they are more harder to find with text editors (for example ctrl + f). IDEs can help though. I guess this is the translate function (t) of the i18next library? Probably not much then what could be done here, as it might be confusing to use another name for it.

Reply:
Yes, "t" it is **the** i18n.t, so kept the same name with the function. One letter funcs/vars are bad, but in this case it clarifies that it is **the** "t" -function.